PMSCore/User.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace PMSCore
+{
+    internal abstract class User
```

Comment:
why do we have this internal abstract class? also shouldn't it inherit from IUser ?